assets/src/room.ts

```diff
@@ -154,25 +153,25 @@ export class Room {
   }
 
   public init = async () => {
-    try {
-      this.localAudioStream = await navigator.mediaDevices.getUserMedia(
-        AUDIO_MEDIA_CONSTRAINTS
-      );
-    } catch (error) {
-      console.error("Error while getting local audio stream", error);
-    }
+    const hasVideoInput: boolean = (
+      await navigator.mediaDevices.enumerateDevices()
+    ).some((device) => device.kind === "videoinput");
```

Comment:
Could we add a comment explaining why we need to enumerateDevices two times? E.g.

```
The first call to enumerateDevices is only to check whether there are any video devices as their ids will not be set until getting permissions from a user.
```